system.go

```diff
@@ -22,6 +22,81 @@ import (
 	"github.com/winlabs/gowin32/wrappers"
 )
 
+type InitiateShutdownFlag uint32

```

Comment:
`InitiateShutdownFlag` -> `InitiateShutdownFlags` for consistency with existing data types.